config/v1/types_infrastructure.go

```diff
@@ -182,6 +182,25 @@ type BareMetalPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// provisioningInterface is the name of the network interface on a Baremetal server connected
+	// to the provisioning network.
+	ProvisioningInterface string `json:"provisioningInterface"`
+
+	// provisioningIp is the VIP to be used by the provisioning service. This must be on the provisioning
+	// subnet, and outside of the DHCP range.
+	ProvisioningIp string `json:"provisioningIp"`
+
+	// dhcpRange is the IP address range on the provisioning subnet from which Baremetal hosts
```

Comment:
indicate format.  I'm assuming CIDR, but I'm not sure.

Reply:
Should this be "ProvisioningDHCPRange" given the Provisioning prefix for the Interface/IP values above?

Reply:
Re the format, the underlying container expects a comma-separated pair of IPs, not a CIDR:

https://github.com/openshift/ironic-image/blob/master/rundnsmasq.sh#L6

We could however convert from a CIDR, either in the container or the MAO if needed?